tests/by-util/test_tr.rs

```diff
@@ -1159,3 +1159,33 @@ fn test_delete_flag_takes_only_one_operand() {
         "extra operand 'p'\nOnly one string may be given when deleting without squeezing repeats.",
     );
 }
+
+#[test]
+fn test_translate_flag_fails_with_more_than_two_operand() {
```

Comment:
The long form of `-t` is `--truncate-set1` and not `--translate`, hence I would use something like:
```suggestion
fn test_truncate_flag_fails_with_more_than_two_operand() {
```